<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginUI</class>
 <widget class="QWidget" name="loginUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(133, 92, 119);</string>
  </property>
  <widget class="QWidget" name="leftSide" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 251, 250);
border-right-color: rgb(27, 0, 21);
</string>
   </property>
   <widget class="QFrame" name="imageFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>401</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:50px;
background-color: rgb(255, 66, 58);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/mainPage.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="titleWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 221, 194);
color:rgb(0,0,0);
border-radius:60px;
</string>
    </property>
    <widget class="QLabel" name="timeTrackingText">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>20</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Time Tracking</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="appText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>App</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="rightSide" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(252, 222, 164);</string>
   </property>
   <widget class="QWidget" name="signUpWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>380</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(125, 193, 83);
color:rgb(255,255,255);
border-radius:60px;
</string>
    </property>
    <widget class="QPushButton" name="signUpButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailInputSignUp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(217, 217, 217, 66);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="emailTextLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="signUpTextLAbel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>20</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="errorTextSignUp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color:red;</string>
     </property>
     <property name="text">
      <string>error text</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameTextLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInputSignUp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(217, 217, 217, 66);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 96, 86);
color:rgb(255,255,255);
border-radius:60px;
</string>
    </property>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailInputLogin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(217, 217, 217, 66);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="emailLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>20</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="errorTextLogin">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color:black;</string>
     </property>
     <property name="text">
      <string>error text</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
